src/js/components/FocusedContainer.js

```diff
@@ -103,12 +105,23 @@ export const FocusedContainer = ({
     if (container) roots.push(container);
 
     // Create and insert focusable nodes to help track when focus
-    // has left this container but without letting focus be noticeably placed
-    // on anything outside the container
+    // has left this container but without letting focus be noticeably
+    // placed on anything outside the container
     if (!hidden && trapFocus) {
       const preDiv = document.createElement('div');
       const postDiv = document.createElement('div');
 
+      preDiv.style.position = 'absolute';
+      preDiv.style.height = '1px';
+      preDiv.style.top = '0';
+      preDiv.style.left = '0';
+      preDiv.style.right = '0';
+      postDiv.style.position = 'absolute';
+      postDiv.style.height = '1px';
+      postDiv.style.left = '0';
+      postDiv.style.right = '0';
+      postDiv.style.bottom = '0';
```

Comment:
I like this better since we are not having to add a keydown event listener.. can we just clean this up a little by abstracting the common styles out

const commonStyles = {
    position: 'absolute',
    height: '1px',
    left: '0',
    right: '0',
};

Object.assign(preDiv.style, {
    ...commonStyles,
    top: '0',
});

same thing for postDiv...

just so if we need to edit anything its in one place ( dont think we ever would) but may be cleaner

Reply:
Yeah agree that would be cleaner, I'll do that

Reply:
Fixed in latest commit

Reply:
Is the 1px needed specifically to ensure it's "in view"?

Reply:
Yeah, if I don't have the 1px it tries to scroll